config/routes.rb

```diff
@@ -4,13 +4,17 @@
     get 'dashboard', to: 'dashboards#index', as: 'dashboard'
     get 'reports', to: 'reports#index', as: 'reports'
     post 'search', to: 'dashboards#search', defaults: { format: :js }
+    get 'display_clinic_finder', to: 'patients#display_clinic_finder', as: 'display_clinic_finder'
+    get 'display_clinic_results', to: 'patients#display_clinic_results', as: 'display_clinic_results'
```

Comment:
We should hook this into `clinicfinders#search` if we can! I bet we can get away without a new endpoint for it.